mobile/src/src/components/Housing/Listings/ListingsContainer.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import { StyleSheet, View, FlatList } from "react-native";
+import ListingView from "./ListingView"
+import { ListingsEndpoint } from "../../../utils/endpoints";
+
+class ListingContainer extends Component{
+
+    state = {
+        listings: [],
+        extendListings: (newListings) => {
+            this.setState({
+                listings: this.state.listings.concat(newListings),
+            });
+        },
+        page: 1,
+        hasMoreListings: true
+    };
+
+    constructor(props) {
+        super(props);
+        this.state.listings = [];
+        this.fetchListings(1, {});
+    }
+
+    fetchListings(page, filters) {
```

Comment:
I suggest we make this function using => syntax. It helps with binding the 'this' keyword to the class, which helps just in case we decide to pass this function as a prop.